src/main/webapp/assets/modules/salary-report/sr-similar-job.js

```diff
@@ -0,0 +1,91 @@
+techlooper.directive("srSimilarJob", function (jsonValue, connectionFactory, $timeout, $translate, validatorService, focus) {
+  return {
+    restrict: "E",
+    replace: true,
+    templateUrl: "modules/salary-report/sr-similar-job.tem.html",
+    link: function (scope, element, attr, ctrl) {
```

Comment:
'ctrl' is defined but never used.<br>'attr' is defined but never used.<br>'element' is defined but never used.